src/common/src/types/chrono_wrapper.rs

```diff
@@ -116,7 +116,7 @@ impl NaiveTimeWrapper {
     pub fn with_secs_nano_value(secs: u32, nano: u32) -> value_encoding::Result<Self> {
         Ok(NaiveTimeWrapper::new(
             NaiveTime::from_num_seconds_from_midnight_opt(secs, nano)
-                .ok_or(ValueEncodingError::InvalidNaiveTimeEncoding(secs, nano))?,
+                .ok_or_else(|| ValueEncodingError::InvalidNaiveTimeEncoding(secs, nano))?,
```

Comment:
There shouldn't be any difference for enum construction.

Reply:
https://github.com/rust-lang/rust-clippy/issues/9663